npeps/npep-230-dry-run.md

```diff
@@ -0,0 +1,62 @@
+# NPEP-95: NPEP template
+
+* Issue: [#230](https://github.com/kubernetes-sigs/network-policy-api/issues/230)
+* Status: Provisional
+
+## TLDR
+
+Add dry-run mode for (B)ANP to allow "disabling" policies without deleting them.
+This should allow implementations add extra audit/monitoring/logging capabilities on top of this feature.
+
+## Goals
+
+A dry-run mode should not affect any connections, and allow the networking plugin to provide feedback 
```

Comment:
Let's be more clear on this:

- A NetworkPolicy object in dry-run mode will not enforce any K8s network policy semantics e.g. it should be treated as if it did not exist. Controllers are free to interpret the contents of the object for purposes of observability/debugging however.